examples/bert/README.md

```diff
@@ -97,14 +96,6 @@ This workflow performs BERT optimization on GPU with CUDA/TensorRT. It performs
     - *PyTorch Model -> Onnx Model -> ONNX Runtime performance tuning with trt_fp16_enable*
     Config file: [bert_trt_gpu.json](bert_trt_gpu.json)
 
-### BERT optimization with TensorRT-Model-Optimizer on CPU/GPU

```

Comment:
Are we removing BERT because it does not have DML deployment support?

Reply:
Yeah. not tested internally